scripts/build-binary

```diff
@@ -59,23 +59,33 @@ function main {
 
 	for os in darwin windows linux; do
 		export GOOS=${os}
-		export GOARCH="amd64"
+		TARGET_ARCHS=("amd64")
 
-		pushd etcd >/dev/null
-			GO_LDFLAGS="-s" ./build
-		popd >/dev/null
-
-		TARGET="etcd-${VER}-${GOOS}-${GOARCH}"
-		mkdir ${TARGET}
-		package ${TARGET} ${PROJ}
-
-		if [ ${GOOS} == "linux" ]; then
-			tar cfz ${TARGET}.tar.gz ${TARGET}
-			echo "Wrote release/${TARGET}.tar.gz"
-		else
-			zip -qr ${TARGET}.zip ${TARGET}
-			echo "Wrote release/${TARGET}.zip"
+		if [[ ${GOOS} == "linux" ]]; then
+			TARGET_ARCHS+=("ppc64le")
 		fi
+
+		for TARGET_ARCH in "${TARGET_ARCHS[@]}"
```

Comment:
this should be `for TARGET_ARCH in "${TARGET_ARCHS[@]}"; do` to match the for loop style above